realsense2_camera/src/t265_realsense_node.cpp

```diff
@@ -42,6 +44,25 @@ void T265RealsenseNode::initializeOdometryInput()
     _use_odom_in = true;
 }
 
+void T265RealsenseNode::setupMapReutilization() {
+    // Load map if configured in the launch file
+    std::string localization_map_filepath;
+    _pnh.param("localization_map_filepath", localization_map_filepath, std::string(""));
+    if (localization_map_filepath.empty())
+    {
+        ROS_INFO("No [localization_map_filepath] specified. No localization data loaded.");
+    }
+    else
+    {
+        this->importLocalizationMap(localization_map_filepath);
+    }

```

Comment:
If an input parameter with a map is provided, load the map. Print an info to stdout otherwise